frontend/src/pages/RunDetails.tsx

```diff
@@ -669,9 +670,18 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
         runFinished = true;
       }
 
-      const workflow = JSON.parse(
-        runDetail.pipeline_runtime!.workflow_manifest || '{}',
-      ) as Workflow;
+      const jsonWorkflow = JSON.parse(runDetail.pipeline_runtime!.workflow_manifest || '{}');
```

Comment:
Thanks! Would it be possible adding a unit test for RunDetails.tsx that goes through this case? So we don't break it accidentally later.

Reply:
Thanks for the review, I will test it.

Reply:
@Bobgy I tested this case in `RunDetails` page and also the `decodeCompressedNodes` in `Utils`. The `kubeflow-pipeline-frontend-test` passed however those tests are now broken: `kubeflow-pipeline-e2e-test`, `kubeflow-pipeline-sample-test` and `kubeflow-pipeline-upgrade-test`.
I am no sure if the reason is related to this PR?